daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/Daffodil.scala

```diff
@@ -458,13 +582,29 @@ class DataProcessor private[sapi] (dp: SDataProcessor)
    *                then no namespace is used. If not preceded by anything,
    *                then Daffodil will figure out the namespace.
    */
-  def setExternalVariables(extVars: Map[String, String]) = dp.setExternalVariables(extVars)
+  def setExternalVariables(extVars: Map[String, String]) =
```

Comment:
Ah, I think this is the one I was seeing not being deprecated. I tihnk I was just confused before. This is the only one that's missing an @deprecated annotation. Everything else looks good.